script/setup

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# script/setup: Set up the application for the first time after cloning, or set
+#               it back to the initial unused state.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+if [ -d vendor/bundle ]; then
+  echo "==> Cleaning installed Ruby dependencies..."
+  git clean -x --force -- vendor/bundle
+fi
+
+echo "==> Bootstrapping..."
+script/bootstrap
+
+if [ ! -f .env.development.local ]; then
+  echo "==> Copying default environment config..."
+  cp .env.example .env.development.local
```

Comment:
This is great - I've often accidentally copied this to `.env` and wondering after running the test suite where my development data has gone!